bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -174,33 +179,53 @@ public BigtableDataGrpcClient(
     this.retryExecutorService = retryExecutorService;
     this.retryOptions = bigtableOptions.getRetryOptions();
 
+    // grpc static variable method descriptors are deprecated.  They are going to be replaced
+    // with static methods.  Once the new static methods exist, then we should use those.
+    // For now, use a hacky approach to get the method descrptors so that all environments work.
+    //
+    // See https://github.com/grpc/grpc-java/issues/1901 for more details.
+    ServerServiceDefinition definition = new BigtableGrpc.BigtableImplBase() {}.bindService();
```

Comment:
Might be slightly nicer to use BigtableGrpc.getServiceDescriptor()

Reply:
BigtableGrpc actually has static getters.  I'm not sure why I missed that.